projs/shadow/shadow-engine/core/inc/log/logger.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
Why do we now have two logging systems?

Reply:
The old thing had a wrapper around spdlog. This might still be useful for showing the logs in the engine in imgui window.